<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewFileWizard</class>
 <widget class="QWizard" name="NewFileWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New File (Resource) to CC3D Simulation</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3" colspan="2">
       <widget class="QLineEdit" name="nameLE"/>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="nameBrowsePB">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>File Type:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="3">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3" rowspan="2" colspan="2">
       <widget class="QComboBox" name="fileTypeCB">
        <item>
         <property name="text">
          <string>Main Python Script</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>XML Script</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Python File</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PIF File</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Concentration File</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="5">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" rowspan="2" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string> Location:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5" rowspan="2">
       <widget class="QPushButton" name="locationBrowsePB">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="2">
       <widget class="QLineEdit" name="locationLE"/>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Project Directory</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3" colspan="2">
       <widget class="QLineEdit" name="projectDirLE">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="3" rowspan="2" colspan="2">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="5" rowspan="3">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="4" rowspan="2">
       <widget class="QLineEdit" name="customTypeLE">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QCheckBox" name="customTypeCHB">
        <property name="text">
         <string> Custom File Type </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>71</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameLE</tabstop>
  <tabstop>nameBrowsePB</tabstop>
  <tabstop>locationLE</tabstop>
  <tabstop>locationBrowsePB</tabstop>
  <tabstop>fileTypeCB</tabstop>
  <tabstop>customTypeCHB</tabstop>
  <tabstop>customTypeLE</tabstop>
  <tabstop>projectDirLE</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>customTypeCHB</sender>
   <signal>toggled(bool)</signal>
   <receiver>customTypeLE</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customTypeCHB</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileTypeCB</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
